### ISSUE TITLE
Ascent Autopilot does not stop gravity turn

### ISSUE THREAD

**Issue by palaslet** [MuMech/MechJeb2#580]
The ascent autopilot does not stop the gravity turn when the AP reach target orbit altitude. I've tried to change the values in the GUI around to see if it was only a GUI bug, but seems deeper.

I'd investigate this myself, but currently having a hard time setting up the project correctly in Visual Studio.

**Comment by gmcnew** [comment]
Please (1) try this with the Kerbal X (a stock ship that we can test too) and (2) add a screenshot of your ascent settings.

**Comment by palaslet** [comment]
This is a bit embarrassing....

I did a lot to try and narrow this down (KerbalX used for all tries). It was reproducible all over the place (removing mods one by one did not help). I even ended up re-installing KSP and only adding MechJeb2-445 build. Keeping all default settings in the GUI (except from warping) and... it worked perfectly!

Then... Trying the settings I used when it failed. And... it failed again.

Looking it over one more time. And... for some reason I've set the orbit altitude to 80.000km instead of 80km. Feeling the pain NASA did with Mars Climate Orbiter...
